pepper-apis/src/test/java/org/broadinstitute/ddp/route/PostPasswordResetRouteTest.java

```diff
@@ -37,7 +36,8 @@
     private static String auth0Domain;
     private static final String testEmail = "test_user@datadonationplatform.org";
     private static final String testRedirectUrl = "http://www.datadonationplatform.org/default-password-reset-page/";
-    private static final String testRedirectUrlWithEmail = testRedirectUrl + "?" + QueryParam.EMAIL + "=" + testEmail;
+    private static final String testRedirectUrlWithEmail = testRedirectUrl + "?" + QueryParam.EMAIL + "="
+            + URLEncoder.encode(testEmail, StandardCharsets.UTF_8);
```

Comment:
Notice here that we didn't test for proper encoding of the email (e.g., the `@` character should be encoded). Because we used `HttpUrl` in the `PostPasswordResetRoute` implementation, it was not encoded. However, now, it is being encoded. it's a breaking change, but I think that's the way it should be -- the query value should be encoded. So I updated the test here to reflect that.